### ISSUE TITLE
大佬你有训练icdar2015这个数据集吗？

### ISSUE THREAD

**Issue by ssvicnent** [qjy981010/CRNN.IIIT-5K.pytorch#4]

**Comment by qjy981010** [comment]
emmm没训过，公开数据集只搞过这个还有一个淘宝的

**Comment by ssvicnent** [comment]
能否加下你的qq向您请教一下问题

**Comment by qjy981010** [comment]
qq号1114943038，不过我很久很久没做过ocr了（，够呛帮的了你

**Comment by FLHonker** [comment]
loss = nan，为何？有bug？
